tests/phpunit/tests/media.php

```diff
@@ -3798,6 +3798,83 @@ function() use ( &$result ) {
 
 		$this->assertFalse( $result );
 	}
+
+	/**
+	 * @ticket 55443
+	 */
+	public function test_wp_print_image_mime_fallback_script_if_content_has_updated_images() {
```

Comment:
Can we have `test_wp_print_image_mime_fallback_shoul_something_something_if_content_has_updated_images`? This makes it clear what the expected behaviour is.

Same for all new test methods in the PR.

Reply:
Updated.